bibernate-core/src/main/java/com/bobocode/petros/bibernate/configuration/BasicDataSourceConfiguration.java

```diff
@@ -11,6 +11,7 @@ public class BasicDataSourceConfiguration implements DataSourceConfiguration {
     private String url;
     private String username;
     private String password;
-    private int poolSize;
+    @Builder.Default
```

Comment:
What would be the behavior in case if data source configuration would be created via no-args constructor instead of builder? Wouldn't it be better to use @Builder(toBuilder = true) instead?

Reply:
discussed